.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish and Tag
```

Comment:
I would suggest to name this file continuous_delivery (so it's the other file is called continuous_integration)

Reply:
Good suggestion. Done.